go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/go-spatial/tegola
 
-go 1.21
```

Comment:
Note, if you rebase against master this should have already been taken care of. Also we use semantic versioning now, so 1.22.5